### PR TITLE
PRs now require testing proof

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
PRs now require evidence of the PR creator having tested their code. This primarily applies to feature, balance, and refactor PRs, which have a high chance of breaking things with relation to the code. For small things that aren't possible to show the difference, such as fixing a runtime, assurance that it doesn't appear anymore is enough. For PRs without testing proof, they will have the [Requires Testing Evidence] label applied to them, blocking the PR from being merged until it is resolved. This will not automatically apply retro-actively for PRs made before this one is merged, but may at a maintainer's discretion.

(For maintainers: Vote passed by majority [here](https://discord.com/channels/596783386295795713/787281082050936832/1033030906522378280))

## How This Contributes To The Skyrat Roleplay Experience
Ensuring people follow the codebase rule of testing things is good.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
It'd go here!
</details>

### CODE REVIEW

**Review by tf-4 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,6 +9,17 @@
...
 <!-- Please add a short description of why you think these changes would benefit the game and the roleplay atmosphere of the server. If you can't justify it in words, it might not be worth adding. -->
 
+## Proof of Testing
+
+<!-- Include any screenshots/videos/debugging steps of the code functioning successfully -->
+
+<details>
+<summary>Screenshots/Videos</summary>
+  
+<!-- Add the proof of testing here, between the </summary> and </details> code blocks -->
```
Comment (GoldenAlpharex) [inline .github/PULL_REQUEST_TEMPLATE.md]: Put this in the same comment block as the one before the summary block.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Zonespace27) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done

**Review by GoldenAlpharex — APPROVED** [review]
Let's go.

**Pull request merged**